addons/account/models/chart_template.py

```diff
@@ -115,6 +115,8 @@ class AccountChartTemplate(models.Model):
         string="Gain Exchange Rate Account", domain=[('internal_type', '=', 'other'), ('deprecated', '=', False)])
     expense_currency_exchange_account_id = fields.Many2one('account.account.template',
         string="Loss Exchange Rate Account", domain=[('internal_type', '=', 'other'), ('deprecated', '=', False)])
+    default_cash_difference_income_account_id = fields.Many2one('account.account.template', string="Cash Difference Income Account")
```

Comment:
rename with property in name

Reply:
look how 'property_account_receivable_id' is set (this one is set on partners, and you property will be on joutnals)